moj-node/server/views/pages/search.html

```diff
@@ -26,6 +26,34 @@
   <![endif]-->
 {% endblock %}
 
+{% block analytics %}
+<!-- Matomo -->
+<script>
```

Comment:
this block should be deleted in this file
it's now `{%block matomo %} which is further down the file`

Reply:
Done